recipes/nvcc/meta.yaml

```diff
@@ -0,0 +1,64 @@
+{% set name = "nvcc" %}
+{% set version = environ.get("CUDA_VER", "9.2") %}
+
+package:
+  name: "{{ name }}"
+
+build:
+  number: 0
+  skip: true  # [not linux]
+
+outputs:
+  - name: "{{ name }}_{{ target_platform }}"
+    version: "{{ version }}"
+    script: install_nvcc.sh
+    build:
+      script_env:
+        - CUDA_HOME
+      ignore_run_exports:
+        - libgcc-ng
+      run_exports:
+        strong:
+          - cudatoolkit {{ version }}|{{ version }}.*
+    requirements:
+      host:
+        # Needed to symlink libcuda into sysroot libs.
+        - {{ compiler("c") }}
+      run:
+        - {{ compiler("c") }}
+        # Host code is forwarded to a C++ compiler
+        - {{ compiler("cxx") }}
+    test:
+      files:
+        - test.cu
+      commands:
+        # Verify the symlink to the libcuda stub library exists.
+        - test -f "$(${CC} --print-sysroot)/lib/libcuda.so"
+        # Verify the activation scripts are in-place.
+        {% for state in ["activate", "deactivate"] %}
+        - test -f "${PREFIX}/etc/conda/{{ state }}.d/{{ PKG_NAME }}_{{ state }}.sh"
+        {% endfor %}
+        # Try using the activation scripts.
+        - source ${PREFIX}/etc/conda/activate.d/{{ PKG_NAME }}_activate.sh
```

Comment:
Can you remove this line? It should have been run already

Reply:
We could, but this makes it clearer to the reader what we are testing. Explicit is better than implicit.

Reply:
But then, we are not testing that `CUDA_HOME` is automatically set. We are checking that `CUDA_HOME` is set when that script is run.

Reply:
I see your point. That said, do these test modifications need to be handled in `staged-recipes` or can they be handled in the feedstock? Note that more work will need to be done in the feedstock anyways.

Reply:
Okay. If you fix the other issue, then I'll merge. Also, please wait for my review on a PR in the feedstock before this package is released.

Reply:
As a higher level comment, making these suggestions is good. Just want to make sure we are progressing on the larger objectives they sit within as well.

I switched things around so now we test activation works without running the activation script first. Also added an explicit test of the activation script after running/testing the deactivation script. This also allowed a little bit of cleanup around the build step. So hopefully a bit nicer. Though we can discuss more in the feedstock.